src/renderer/api/kube-api.ts

```diff
@@ -8,10 +8,17 @@ import { apiKube } from "./index";
 import { kubeWatchApi } from "./kube-watch-api";
 import { apiManager } from "./api-manager";
 import { createKubeApiURL, parseKubeApi } from "./kube-api-parse";
-import { apiKubePrefix, isDevelopment } from "../../common/vars";
+import { apiKubePrefix, isDevelopment, isTestEnv } from "../../common/vars";
 
 export interface IKubeApiOptions<T extends KubeObject> {
   apiBase?: string; // base api-path for listing all resources, e.g. "/api/v1/pods"
+
+  // If the API uses a different API endpoint (e.g. apiBase) depending on the cluster version,
```

Comment:
Maybe use `tsdoc` format here (would appear in generated extension api docs)?

Reply:
Fixed, although I'm not confident it will end up in the API Docs (it seems the interface is not listed there, just referenced).

I also didn't find clear information if documenting interfaces is even supported in TSDoc at the moment.